crates/toml_datetime/src/lib.rs

```diff
@@ -9,6 +9,7 @@
 // and lets them ensure that there is indeed no unsafe code as opposed to
 // something they couldn't detect (e.g. unsafe added via macro expansion, etc).
 #![forbid(unsafe_code)]
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
Could you instead switch these to ` #![cfg_attr(docsrs, feature(doc_auto_cfg))] ` and remove the `doc(cfg(...))` attributes?

Reply:
i'm staying away from it personally until it's stable but sure, updated